ca/certificate-authority.go

```diff
@@ -125,6 +125,14 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
+	numPolicies := 0
+	for _, v := range cfsslConfigObj.Signing.Profiles {
```

Comment:
I think we want to check that *each* profile contains at least one profile, right?